### PR TITLE
Article milestone2

### PR DESCRIPTION
**Pull request by blakbox23**
We have worked on the Second milestone for "tnw-news" project;
We build 3 remaining sections (Categories, Latest NEWS, Latest Deals) for three screens (Mobile, Tablet and laptop).
Kindly Review our code.
Note: The (Header, Footer, Highlights, and Latest Funding) Sections were implemented in milestone 1
Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -116,56 +117,133 @@ <h1>Latest news ></h1>
...
-        <div class="latest-news-image">
-          <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png" alt="latest-news">
+          <div class="latest-news-image">
+            <img
+              src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png"
+              alt="latest-news"
+            />
+          </div>
+          <div class="latest-news-text flex">
+            <p class="latest-news-bg-text">
```
Comment (henatan99) [inline index.html]: - Change the p-tag to proper hx - tag and also add a-tag for this element. Extrapolate this comment to other latest-news articles also

File: index.html [hunk 2]
```diff
@@ -321,6 +401,372 @@ <h1>Latest funding rounds on index.co</h1>
...
+          <div class="categories-item-title">
+            <h1><a href="#">Hard fork ></a></h1>
+          </div>
+          <div class="categories-image-section cat-one flex">
+            <div class="categories-image-text flex">
+              <p class="categories-mostpop">MOST POPULAR</p>
+              <p class="categories-mostpop-title">
+                GameStop stocks hit record high — thanks to Redditors
+              </p>
+              <p class="categories-mostpop-footer">Ivan Mehta 1 day ago</p>
```
Comment (henatan99) [inline index.html]: - Change this p-tag into appropriate hx - tag and also warp it with or embed in a-tag to this element. Please extrapolate this comment to the other categories too.

File: index.html [hunk 3]
```diff
@@ -321,6 +401,372 @@ <h1>Latest funding rounds on index.co</h1>
...
+            </div>
+          </div>
+
+          <div class="categories-latest-section flex">
+            <div class="categories-latest-section-text">
+              <p class="categories-latest-txt1">LATEST</p>
+              <p class="categories-latest-txt2">
```
Comment (henatan99) [inline index.html]: - Change this p-tag to proper hx-tag and also add a-tag for linking. Please extrapolate this comment to the other similar articles in categories section too.

File: index.html [hunk 4]
```diff
@@ -321,6 +401,372 @@ <h1>Latest funding rounds on index.co</h1>
...
+                nearly 50%
+              </li>
+              <li>
+                CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price
+                nearly 50%
+              </li>
+              <li>
+                CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price
+                nearly 50%
+              </li>
```
Comment (henatan99) [inline index.html]: - Al the list elements here should have embedded a-tag. This comment works for the other list elements in other ul wrappers in categories section

File: index.html [hunk 5]
```diff
@@ -321,6 +401,372 @@ <h1>Latest funding rounds on index.co</h1>
...
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="latest-deals">
+      <div class="latest-deals-title flex">
+        <h1><a href="#">Latest deals ></a></h1>
+        <p>See all details</p>
```
Comment (henatan99) [inline index.html]: -This element should also have a - tag

File: index.html [hunk 6]
```diff
@@ -321,6 +401,372 @@ <h1>Latest funding rounds on index.co</h1>
...
+          <div class="latest-deals-image">
+            <img
+              src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp2.stackassets.com/7a5a98dff30d5c38ecbc1700812eb2e1b93dce93/store/4b1f63289bd195a7bbafafbe36579fe597a5f1e7a0cb8165012908f3db74/sale_28088_primary_image.jpg"
+              alt="latest-deals"
+            />
+          </div>
+          <div class="latest-deals-text flex">
+            <p class="latest-deals-bg-text">
+              Help kill the ‘Bernie sitting’ image generator
+            </p>
```
Comment (henatan99) [inline index.html]: -This p-tag should be a proper hx-tag and add link to it (a-tag). This comment applies to the other articles in the latest-deal section.

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
Hi there, you did amazing work on this project. :clap: :clap: But you need to work on some mandatory improvements before your project gets approved. Please check the comments below.

### Reminder: This is your first review and you will have a maximum of only two more reviews (re-reviews) left. Please make sure you make all the requested changes before re-submission. 
## To Highlight
- Nice project organization 
- You have made a very nice clone of the original page 
- Very nice responsive design 

## To Improve [MANDATORY]
- Apply semantic tags for article elements 
- Please use proper hx-tags and a-tags in article elements of both the latest news, categories, and latest sections. See the inline comments 
- You forgot the yellow triangle price tag element in the latest deals articles on tablet and desktop views

### Please make the requested changes and submit for re-review. Be reminded that you have a maximum of two more reviews and if your project is not approved in the third review you will repeat the project. 

### Happy coding :computer: :computer_mouse:

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved image

Congratulations @blakbox23  and @ahzia for such hard work your project looks amazing it speaks by itself how much you worked hard and you implemented all changes requested by the previous TSE

### Optional
Reduce padding between the highlights section and the latest news on the big screen it doesn't look good

## Kudos once again feel free to merge and move on to the next project

All the best,

Lydia Ingabire

**Pull request merged**